src/index.ts

```diff
@@ -23,8 +25,9 @@ export = class extends Composer implements RNS {
    */
   constructor (public web3: Web3, options?: Options) {
     super(web3, options);
-    this._subdomains = new Subdomains(this.web3, options);
-    this._resolutions = new Resolutions(this.web3, options);
+    this._utils = new Utils();
+    this._subdomains = new Subdomains(this.web3, this._utils, options);
+    this._resolutions = new Resolutions(this.web3, this._utils, options);
```

Comment:
Following comment on the class, this two classes would not need to instance an utils. Just using it.